### PR TITLE
Add log when demangling is complete

### PR DESCRIPTION
**Pull request by nihaals**
I'm not sure if there's any difference between `print()` and `log_info()` but I saw `log_info()` in an official plugin so went for that. I'm also not sure if there's a better way to indicate that the process is complete but at the moment there isn't any real indication and it's hard to tell if it's complete or not.

### CODE REVIEW

File: __init__.py
Comment (saagarjha) [inline __init__.py]:
```suggestion
	binaryninja.log_info("Swift demangling complete!")
```

If you want to go above and beyond I'd also add a bit of detail about how much work was done, e.g.

```suggestion
	binaryninja.log_info(f"Swift demangling complete! Updated {len(swift_functions)} functions and {len(swift_variables)} variables.")
```

**Review by saagarjha — APPROVED** [review]
lgtm, but consider the following change.

File: __init__.py
Comment (nihaals) [reply __init__.py]: That also just seems like potentially useful information, good idea

**Review by saagarjha — APPROVED** [review]
lgtm, do you mind if I squash this into one commit before merging?

**Comment by nihaals** [comment]
Squashed!

**Comment by saagarjha** [comment]
Thanks!

**Pull request merged**
